package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "nodemon server/www.js --exec babel-node --presets es2015,stage-2",
     "lint": "eslint .",
-    "test": "npm run lint && mocha",
+    "test": "mocha --compilers js:babel-core/register",
```

Comment:
Same with all the other scripts. You are currently always using the global version, not the local version from your `node_modules` folder.